osi_object.proto

```diff
@@ -436,6 +437,84 @@ message MovingObject
         // through.
         //
         optional double ground_clearance = 6;
+
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 7;
+
+        // \brief Detailed wheel data.
+        // The focus is on the description of a wheel regarding the perceivable 
+        // information from the outside. 
+        // It is not intended to be used for e.g. dynamic calculations.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+            
+            // Position of the wheel (geometric center of the wheel).
```

Comment:
Does this line mean the position is measured from the center of the wheel?

Reply:
yes, more precisely when there is no load on the wheel, since that could shift it's center position and shape.

Reply:
@paagkame good point, so I think it would be the vector from center of bounding box to the center of the wheel. I think it would make sense to write a vector description like in line 418ff. of osi_object.proto and maybe explain the center of the wheel better and the implications as @kmeids attempts. I guess also steering might change the center of the wheel position. So I think it has to be clear that this vector will not have a constant value. (so I hope I have the correct understanding here).

Reply:
I changed the description. Please give some feedback @kmeids , @clemenshabedank , @paagkame

Reply:
Ok I think the description should be precise now, if it is supposed to be a not-changing field.

Reply:
We discussed the point now and did a small adaption. It's ok now.